mirgecom/profiling.py

```diff
@@ -113,8 +122,6 @@ def __del__(self):
         self.profile_results.clear()
```

Comment:
Probably a warning here, that cloned instances have spurious stats collection?

Reply:
Just as a comment, right?

Reply:
> Just as a comment, right?

Meant to say warn via `warnings.warn`.